engine/utils/String.cpp

```diff
@@ -283,6 +283,26 @@ String String::FromLong(const long& value)
     return GetFromFormat("%ld", value);
 }
 
+String String::FromSizeT(const size_t& value)
+{
+    return GetFromFormat("%lu", value);
+}
+
+String String::FromU32(const uint32_t& value)
+{
+    return GetFromFormat("%lu", value);
+}
+
+String String::FromU64(const uint64_t& value)
+{
+    return GetFromFormat("%lu", value);
+}
+
+Hash::StringId String::WSID(const String& str)
+{
+    return Hash::WSID(str.Str());
+}
+
```

Comment:
Same comment as previous PR: `GetFromFormat` does the job fine and `WSID` can remain a free floating function.

Reply:
The rebases keep reverting this. Next time we do this we MUST do the PRs in order otherwise we'll have to deal with this type of shenanigans again.

Reply:
Done.